connectivity/FEATURE_BLE/source/common/ble_trace_helpers.cpp

```diff
@@ -0,0 +1,123 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2021 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ble_trace_helpers.h"
+
+namespace ble {
+
+const char* dm_callback_event_to_string(uint8_t event) {
+    const char* ret = "INVALID";
```

Comment:
Is it invalid or an unsupported event ?

Reply:
I don't know, this basically means the traces need updating because the stack added new values to the enum

Reply:
I called in invalid because it's a value outside the enum - hence invalid